src/gsi-ncdiag/gsi_ncdiag.py

```diff
@@ -137,7 +137,7 @@
     'SCCF_chan_wavelen': ('sensorCentralFrequency', 'double'),
     'QI_with_FC': ('percentConfidenceWithForecast', 'float'),
     'QI_without_FC': ('percentConfidenceWithoutForecast', 'float'),
-    'LaunchTime': ('LaunchTime', 'float'),
+    'releaseTime': ('releaseTime', 'integer'),
```

Comment:
believe the left hand side is the `GSI` variable name so should remain as `LaunchTime` but am not entirely sure